test_scripts/Defects/8_2/2468_1_AddCommand_Both_UI_Timeout.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------------
+-- Use case: AddCommand
+-- Item: Timeout from UI portion of request
+--
+-- Requirement summary:
+-- [AddCommand] GENERIC_ERROR: getting GENERIC_ERROR on UI.AddCommand timeout
+--
+-- Description:
+-- Mobile application sends valid AddCommand request with both "vrCommands"and "menuParams" 
+-- data and gets "SUCCESS" for VR.AddCommand and no response for UI.AddCommand from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+
+-- Steps:
+-- 1. appID requests AddCommand with both vrCommands and menuParams
+-- 2. SDL transfers the UI part of request with allowed parameters to HMI
+-- 3. SDL transfers the VR part of request with allowed parameters to HMI
+-- 4. SDL receives VR part of response from HMI with "SUCCESS" result code
+-- 5. SDL does not receive UI part of response
+
+-- Expected:
+-- SDL responds with (resultCode: GENERIC_ERROR, success: false) to mobile application
+-- SDL sends UI.DeleteCommand and VR.DeleteCommand based on the original request
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  cmdID = 11,
+  menuParams = {
+    position = 0,
+    menuName = "Commandpositive",
+    secondaryText = "Secondary",
+    tertiaryText = "Tertiary"
+  },
+  vrCommands = {
+    "VRCommandonepositive",
+    "VRCommandonepositivedouble"
+  },
+  cmdIcon = {
+    value = "icon.png",
+    imageType = "DYNAMIC"
+  },
+  secondaryImage = {
+    value = "icon.png",
+    imageType = "DYNAMIC"
+  }
+}
+
+local responseUiParams = {
```

Comment:
Should this be renamed to 
```suggestion
local requestUiParams = {
```
given that its used with ExpectRequest

Reply:
This comment applies to all other scripts (with UI.AddCommand) as well